lib/oppen/print_stack.rb

```diff
@@ -21,59 +21,59 @@ def initialize(offset, break_type)
     # IO element that builds the output.
     attr_reader :buffer
 
-    # Config containing customization flags
+    # Config containing customization flags.
     attr_reader :config
 
-    # Callable that generate spaces
+    # Callable that generate spaces.
     attr_reader :genspace
 
     # Array representing the stack of PrintStackEntries.
     attr_reader :items
 
-    # Delimiter between lines in output
+    # Delimiter between lines in output.
     attr_reader :new_line
 
-    # Maximum allowed width for printing (Called length in the original paper).
-    attr_reader :width
-
     # Current available space (Called index in the original paper).
     #
     # @return [Integer] Current available space (Called index in the original paper).
     attr_reader :space
 
+    # Maximum allowed width for printing (Called length in the original paper).
+    attr_reader :width
+
     def initialize(width, new_line, config, space, out)
       @buffer = out
       @config = config
       @genspace =
-        if space.respond_to?(:call)
+        if space.respond_to? :call
           raise ArgumentError, 'space argument must be a Proc of arity 1' \
             if space.to_proc.arity != 1
 
           space
         else
           ->(n) { space * n }
         end
-      @indent = 0
+      @indent = 0 # the amount of indentation to display on the next non empty new line.
       @items = []
       @new_line = new_line
       @width = width
       @space = width
     end
 
-    # Returns the output of the print stack
+    # Returns the output of the print stack.
     #
     # @return [String]
     def output
-      buffer.truncate(buffer.pos)
+      buffer.truncate buffer.pos
       buffer.string
     end
 
     # Core method responsible for building the print stack and the output string.
     #
     # @note Called Print in the original paper.
```

Comment:
```suggestion
    # @note Called `Print` in the original paper.
```

Please review all occurences of _"called xxx"_ in the docs, and make sure that technical names get a technical treatment with backticks.